### ISSUE TITLE
Can't Send an event as the example setup show error 500 only post request is allowed

### ISSUE THREAD

**Issue by Altroo** [RivercodeAB/facebook-conversion-api-nextjs#15]
/api/fb-event.js

```
import { fbEventsHandler } from '@rivercode/facebook-conversion-api-nextjs/handlers';

export default fbEventsHandler;
```
_app.tsx

```
import { FBPixelScript, FBPixelProvider } from '@rivercode/facebook-conversion-api-nextjs/components';

<ThemeProvider theme={getDefaultTheme()}>
	{/* CssBaseline kickstart an elegant, consistent, and simple baseline to build upon. */}
	<CssBaseline />
	<SessionProvider
		// Provider options are not required but can be useful in situations where
		// you have a short session maxAge time. Shown here with default values.
		session={session}>
		<InitContextProvider>
			<CustomContainer>
				<FBPixelScript />
				<FBPixelProvider>
					<Component {...pageProps} />
				</FBPixelProvider>
			</CustomContainer>
		</InitContextProvider>
	</SessionProvider>
</ThemeProvider>
```

index.tsx

```
const onClickHandler = () => {
  fbEvent({
	  eventName: 'AddToCart', // ViewContent, AddToCart, InitiateCheckout or Purchase
	  // eventId: 'eventId', // optional, unique event id's will be generated by default
	  emails: ['test@email.com'], // optional
	  phones: ['061122334455'], // optional
	  products: [
		  {
			  sku: 'product123',
			  quantity: 1,
		  },
	  ],
	  value: 1000,
	  currency: 'USD',
	  enableStandardPixel: true,
  });
  };
};
```
.env

```
FB_ACCESS_TOKEN=XXXX
NEXT_PUBLIC_FB_PIXEL_ID=XXXX
NEXT_PUBLIC_FB_DEBUG=true
```

Console output
```
------------------------------------ 
Facebook Conversion API Debugging 
------------------------------------ 
Client Side Event: PageView 
Comme certains cookies utilisent incorrectement l’attribut « SameSite », le fonctionnement ne sera pas celui attendu. 5
Client Side Event: AddToCart 
Client Side Payload: {"content_type":"product","contents":[{"id":"product123","quantity":1}],"value":1000,"currency":"USD"} 
Client Side Event ID: 3c1fab02-e567-4d69-8827-9307f8cae6a4 
XHRPOSThttp://localhost:3000/api/fb-events
[HTTP/1.1 500 Internal Server Error 6610ms]

[Fast Refresh] rebuilding 2 
Server Side Event: AddToCart (500) 
Server Side Payload: {"eventName":"AddToCart","eventId":"3c1fab02-e567-4d69-8827-9307f8cae6a4","emails":["test@email.com"],"phones":["061122334455"],"products":[{"sku":"product123","quantity":1}],"value":1000,"currency":"USD","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/110.0","sourceUrl":"http://localhost:3000/"} 
```

Tried both local & remote server, always 500, on chrome it shows : 

````
POST http://localhost:3000/api/fb-events 500 (Internal Server Error)     
eval @ instrument.js?b344:137
eval @ conversion-api.js?b48a:47
sentryWrapped @ helpers.js?0b5b:90
setTimeout (asynchrone)
eval @ trycatch.js?8591:105
fbEvent @ conversion-api.js?b48a:34
onClickHandler @ index.tsx?07ff:31
callCallback @ react-dom.development.js?ac89:4164
sentryWrapped @ helpers.js?0b5b:90
invokeGuardedCallbackDev @ react-dom.development.js?ac89:4213
invokeGuardedCallback @ react-dom.development.js?ac89:4277
invokeGuardedCallbackAndCatchFirstError @ react-dom.development.js?ac89:4291
executeDispatch @ react-dom.development.js?ac89:9041
processDispatchQueueItemsInOrder @ react-dom.development.js?ac89:9073
processDispatchQueue @ react-dom.development.js?ac89:9086
dispatchEventsForPlugins @ react-dom.development.js?ac89:9097
eval @ react-dom.development.js?ac89:9288
batchedUpdates$1 @ react-dom.development.js?ac89:26140
batchedUpdates @ react-dom.development.js?ac89:3991
dispatchEventForPluginEventSystem @ react-dom.development.js?ac89:9287
dispatchEventWithEnableCapturePhaseSelectiveHydrationWithoutDiscreteEventReplay @ react-dom.development.js?ac89:6465
dispatchEvent @ react-dom.development.js?ac89:6457
dispatchDiscreteEvent @ react-dom.development.js?ac89:6430
sentryWrapped @ helpers.js?0b5b:90
instrument.js?b344:108 Server Side Event: AddToCart (500)
instrument.js?b344:108 Server Side Payload: {"eventName":"AddToCart","eventId":"4ce4ee3b-0f3e-4a45-999f-3e7a3275d4cf","emails":["test@email.com"],"phones":["061122334455"],"products":[{"sku":"product123","quantity":1}],"value":1000,"currency":"USD","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36","sourceUrl":"http://localhost:3000/"}
```

**Comment by Altroo** [comment]
It's like client side is sent, but not the server side.

<img width="675" alt="Capture d’écran 2023-01-30 à 17 37 52" src="https://user-images.githubusercontent.com/30314254/215539052-82b5a2d6-39db-46d4-a436-07242cb240f1.png">

**Comment by ovdahlberg** [comment]
Are you running Node 18? @Altroo

Then it's probably related to: https://github.com/RivercodeAB/facebook-conversion-api-nextjs/issues/14

**Comment by ovdahlberg** [comment]
Hi @Altroo!

This should be fixed in 4.3.1. Please try and see if it works for you now :)
